dispatcher/dispatcher.go

```diff
@@ -88,7 +88,7 @@ type CallbackHandler interface {
 	OnNodeFail(nodeid NodeId)
 	OnNodeRecover(nodeid NodeId)
 
-	// Notifies that the node's UART was written with data.
+	// OnUartWrite notifies that the node's UART was written with data.
```

Comment:
This is the documentation of a method in an interface; and GoLand IDE has a checking rule for such cases that the comment starts with the name of the method. This seemed recommended a while ago (e.g. 2016: https://stackoverflow.com/questions/35591301/function-name-in-comment) but since then has been removed again. GoLand is still recommending it.  I can try not to use it or revert.

Reply:
I thought I have removed this comment.